### ISSUE TITLE
Opening large dta file in Stata 13.1

### ISSUE THREAD

**Issue by benjamin-chan** [sjewo/readstata13#20]
I wrote a large file using `readstata13_0.7`. The dta file was written without error. However, when I try to open the file from Stata 13.1, I get the following message:

```
. use "E:\Share\DataRepository\Colorado Medicaid\Staged\rxClaims_Colorado.dta", clear
(Written by R)
.dta file corrupt
    The file does not record <strls> where expected.  Either the file was written incorrectly in the first place or the file has become corrupted.
r(612);
```

The file is 3 GB. I've tried opening a smaller version of the data file (a 5% sample of rows, same columns, 150 MB) and it opened without error.

The call to `save.dta13` that saved the dta file was:

```
save.dta13(rxClaims, file=sprintf("%s.dta", f), convert.dates=TRUE, compress=TRUE)
```

System information:

```
##                      sysname                      release 
##                    "Windows"            "Server 2008 x64" 
##                      version                     nodename 
## "build 7601, Service Pack 1"                       "CHSE" 
##                      machine                        login 
##                     "x86-64"                      "chanb" 
##                         user               effective_user 
##                      "chanb"                      "chanb"
```

Session information:

```
## R version 3.1.2 (2014-10-31)
## Platform: x86_64-w64-mingw32/x64 (64-bit)
## 
## attached base packages:
## [1] stats     graphics  grDevices utils     datasets  methods   base     
## 
## other attached packages:
##  [1] lubridate_1.3.3  xlsx_0.5.7       xlsxjars_0.6.1   rJava_0.9-6     
##  [5] sas7bdat_0.5     readstata13_0.7  foreign_0.8-62   data.table_1.9.4
##  [9] chse_0.6         DiagrammeR_0.8  
## 
## loaded via a namespace (and not attached):
##  [1] chron_2.3-45    digest_0.6.8    evaluate_0.5.5  formatR_1.0    
##  [5] htmltools_0.2.6 htmlwidgets_0.5 jsonlite_0.9.16 knitr_1.9      
##  [9] magrittr_1.5    memoise_0.2.1   plyr_1.8.1      Rcpp_0.11.6    
## [13] reshape2_1.4.1  rmarkdown_0.5.1 rstudioapi_0.2  stringi_0.5-5  
## [17] stringr_1.0.0   tools_3.1.2     yaml_2.1.13
```

**Comment by JanMarvin** [comment]
Hey @benjamin-chan ,

can you please check this with the latest release? [1] Sounds to me like this is just what we fixed in 0.7.1. It is on CRAN, just run `install.packages("readstata13")`.
You have to rewrite the file using readstata13. Importing this in Stata is not possible (they check the map positions, if the position does not look like what they expect, they bail out). The dta-file is correct apart from the map position.
Sorry for the inconvenience.

Marvin

[1] https://github.com/sjewo/readstata13/releases/tag/0.7.1

**Comment by benjamin-chan** [comment]
installing 0.7.1 seemed to work
